doc/nrf/libraries/caf/settings_loader.rst

```diff
@@ -0,0 +1,62 @@
+.. _caf_settings_loader:
+
+CAF: Settings loader module
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Zephyr's Bluetooth® stack does not load the :ref:`zephyr:settings_api` data on its own.
+Zephyr assumes that the application will call :c:func:`settings_load` after completing all necessary initialization.
+The |settings_loader| of the :ref:`lib_caf` (CAF) is responsible for loading the data from non-volatile memory after all necessary initialization is done.
```

Comment:
Currently it suggests that settings module is used to load Bluetooth settings which is partially true (our modules also rely on settings). I would prefer to inform about loading settings for all settings users explicitly mentioning that settings users are among others application modules (from CAF or user-specific).

Reply:
I will simplify this to be as simple as possible. I would like to avoid mentioning explicit modules at this point as this is just an general overview of what module does.